client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -16,7 +16,8 @@ import {
 	getImageEditorTransform,
 	getImageEditorFileInfo,
 	getImageEditorCrop,
-	isImageEditorImageLoaded
+	isImageEditorImageLoaded,
+	getImageMeetsMinimumDimensions
```

Comment:
Maybe let's wait for a few other folks to chime in on ^^ before you start refactoring though 😄